src/globals/scss/_theme.scss

```diff
@@ -0,0 +1,211 @@
+  $support-03: $color__yellow-30 !default !global;
+  $support-04: $color__blue-30 !default !global;
+  $nav-01: $color__navy-gray-1 !default !global;
+  $nav-02: $color__blue-90 !default !global;
+  $nav-03: $color__purple-30 !default !global;
+  $nav-04: $color__purple-60 !default !global;
+  $nav-05: $color__teal-40 !default !global;
+  $nav-06: $color__teal-50 !default !global;
+  $nav-07: $color__blue-30 !default !global;
+  $nav-08: $color__blue-51 !default !global;
+
+  $hover-primary: darken($brand-01, 10%) !default !global;
+  $hover-primary-text: darken($brand-01, 15%) !default !global;
+  $hover-danger: darken($support-01, 10%) !default !global;
+  $hover-secondary: $brand-01 !default !global;
+  $hover-row: rgba($brand-02, 0.1) !default !global;
+
+  // Global
+  $input-border: 1px solid transparent !default !global;
+  $input-label-weight: 600 !default !global;
+
+  // Button Theme Variables
+  $button-font-weight: 600 !default !global;
+  $button-font-size: 0.875rem !default !global;
+  $button-border-radius: 0 !default !global;
+  $button-height: 40px !default !global;
+  $button-padding: 0 $spacing-md !default !global;
+  $button-padding-sm: 0 $spacing-xs !default !global;
+  $button-border-width: 2px !default !global;
+
+  // Accordion (Reverse)
+  $accordion-flex-direction: row !default !global;
+  $accordion-justify-content: flex-start !default !global;
+  $accordion-arrow-margin: 0 0 0 $spacing-2xs !default !global;
+  $accordion-title-margin: 0 0 0 $spacing-md !default !global;
+  $accordion-content-padding: 0 $spacing-md 0 $spacing-2xl !default !global;
+
+  // Card
+  $card-text-align: center !default !global;
+  $card-flex-align: center !default !global;
+
+  // Checkbox
+  $checkbox-border-width: 2px !default !global;
+
+  // Code Snippet
+  $snippet-background-color: $ui-01 !default !global;
+  $snippet-border-color: $ui-03 !default !global;
+
+  // Content Switcher
+  $content-switcher-border-radius: 8px !default !global;
+  $content-switcher-option-border: 1px solid $brand-01 !default !global;
+
+  // Data Table
+  $data-table-heading-transform: uppercase !default !global;
+  $data-table-heading-border-bottom: 1px solid $brand-01 !default !global;
+  $data-table-row-height: 2rem !default !global;
+
+  // Modal
+  $modal-border-top: $brand-01 4px solid !default !global;
+  $modal-footer-background-color: $ui-02 !default !global;
+
+  // Progress Indicator
+  $progress-indicator-bar-width: 1px inset transparent !default !global;
+  $progress-indicator-stroke-width: 5 !default !global;
+  $progress-indicator-line-offset: 0.625rem !default !global;
+
+  // Radio Button
+  $radio-border-width: 2px !default !global;
+
+  // Structured Theme Variables
+  $structured-list-padding: 2rem !default !global;
+  $structured-list-text-transform: none !default !global;
+
+  // Skeleton Loading
+  $skeleton: rgba($color__blue-51, 0.1) !default !global; //old $field-01
+}
+
+// Light Experimental Theme
+@mixin theme--experimental() {
```

Comment:
Would it be helpful to just call this in the experimental mixins themselves instead of toggling at the top-level?

Reply:
where do you mean exactly?